.travis.yml

```diff
@@ -11,7 +11,7 @@ branches:
     - master
 cache: false
 before_deploy:
-  - npm run compile:prod
+  - TERRA_DEV_SITE_NEW_RELIC_LICENSE_KEY='c494ac44c8' TERRA_DEV_SITE_NEW_RELIC_APPLICATION_ID='141260567' npm run compile:prod
```

Comment:
Should these values be set in  =travis-ci.org dashboard like the $GITHUB_TOKEN ?

Reply:
We can expose these, they'd be exposed regardless when the JS is executed in the front end. I added them here this way so that they are only used for this webpack deployment build and not for any of the tests runs.